TSPL.docc/ReferenceManual/Declarations.md

```diff
@@ -980,6 +1044,117 @@ see <doc:Functions#In-Out-Parameters>.
   ```
 -->
 
+#### Borrowing and Consuming Parameters
+
+The Swift compiler has varying ownership rules that it
+can use to manage object lifetimes across function calls.
+The default rules work well for most cases,
+but if necessary, you can use one of the `borrowing`
+or `consuming` modifiers to choose a specific ownership
+rule for a particular argument.
+
+Unlike `inout`, neither `borrowing` nor
+`consuming` parameters require any special
+notation when you call the function:
+
+```swift
+func someFunction(a: borrowing A, b: consuming B) { ... }
+
+someFunction(a: someA, b: someB)
+```
+
+The `borrowing` modifier indicates that the function
+does not expect to keep the value.
```

Comment:
Let's rephrase this as a more rigorous definition.  As written, it suggests that a function might sometimes keep a borrowed value, possibly "unexpectly" and without copying.

This sounds like it might be a usage guideline — you typically write `borrowing` when a function uses a value only during the function call, without any need to keep the value for later.

Reply:
Good point.  Changed "does not expect to" to "will not" (which is accurate).